cg/meta/compress/compress.py

```diff
@@ -81,8 +81,10 @@ def compress_fastq(self, sample_id: str) -> bool:
                 )
                 all_ok = False
                 continue
-
-            if not self.crunchy_api.is_fastq_compression_possible(compression_obj=compression):
+            is_spring_archived: bool = self._is_spring_archived(compression)
+            if not self.crunchy_api.is_fastq_compression_possible(
```

Comment:
consider extracting this call to a separate variable, I think it is more readable that way